webpack.config.js

```diff
@@ -4,6 +4,7 @@
 const path = require( 'path' );
 const process = require( 'process' );
 const { fromPairs } = require( 'lodash' );
+const SVGSpritemapPlugin = require( 'svg-spritemap-webpack-plugin' );
```

Comment:
I was between https://github.com/JetBrains/svg-sprite-loader and https://github.com/cascornelissen/svg-spritemap-webpack-plugin. And although the first one is a JetBrains repo with more downloads, the second one seems to be more maintained.